programs/loft.sc

```diff
@@ -0,0 +1,356 @@
+__config() -> {
+    'stay_loaded'->true,
+    'commands'->{
+        'wand'->['wand'],
+        'paste <Block> <replace>'->['paste'],
+        'paste <Block>'->['paste',null],
+        'paste'->['paste','diamond_block',null],
+        'undo'->['undo']
+    },
+    'arguments'->{
+        'Block'->{'type'->'term', 'suggest'->[
+			'white_concrete','yellow_concrete','orange_concrete','purple_concrete',
+			'lime_concrete','red_concrete','green_concrete','magenta_concrete',
+			'brown_concrete','gray_concrete','light_gray_concrete','blue_concrete',
+			'light_blue_concrete','pink_concrete','cyan_concrete','black_concrete'
+			]
+		},
+        'replace'->{'type'->'term','suggest'->['air','water','lava']}
+    }
+};
+
+global_tool='phantom_membrane';
+global_block='diamond_block';
+global_replace=null;
+global_sets=[]; //array containing current information about the m splines created
+global_points=[]; //vector of maximum n points of the current spline that is being created by the player
```

Comment:
No need to set this here if you are gonna set it again inside `__on_player_breaks_block`. But maybe it's good to have it here so someone reading the code knows about all the global variable syou are gonna use.

Reply:
So, can global variables be declared in one function of the script?